Assets/MRTK.Services/InputSystem/GazeProvider.cs

```diff
@@ -81,13 +81,8 @@ public bool Enabled
             set { enabled = value; }
         }
 
-        [SerializeField]
-        [Tooltip("If true, eye-based tracking will be used when available. Requires the 'Gaze Input' permission and device eye calibration to have been run.")]
-        [Help("When enabling eye tracking, please follow the instructions at "
-               + "https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/EyeTracking/EyeTracking_BasicSetup.html#eye-tracking-requirements "
-               + "to set up 'Gaze Input' capabilities through Visual Studio.", "", false)]
         [FormerlySerializedAs("preferEyeTracking")]
-        private bool useEyeTracking = true;
+        private bool useEyeTracking;
```

Comment:
if this used to be serialized here we should add an entry to the release notes for the next version and mark this as a breaking change so users know that they now have to change this in their profile rather than the gaze provider itself.